xgcm/grid.py

```diff
@@ -56,7 +59,15 @@ class Axis:
     differentiated by their length.
     """
 
-    def __init__(self, ds, axis_name, periodic=True, default_shifts={}, coords=None):
+    def __init__(
+        self,
+        ds,
+        axis_name,
+        periodic=True,
+        default_shifts={},
+        coords=None,
```

Comment:
Do wee need a `fill_value` here too?

Reply:
hmm.. I guess. So like boundary this could be a single value or a dict mapping axis to value?

Reply:
To me that seems the most consistent, yes.